### ISSUE TITLE
Isort `multi_line_output = 3` conflicts with C812

### ISSUE THREAD

**Issue by user-cjw** [wemake-services/wemake-python-styleguide#2523]
### What's wrong

Using `multi_line_output = 3` as suggest in documentation, running Isort on file will sort import as
```python
from mypackage.storage.backend import (
    get_master,
    get_max_record_id,
    user_role_table,
    search
)
```
Since there is no comma after last import, `C812 missing trailing comma` will be raised on line 5. 

If I add a comma after that line, two conflicts will be reported instead: `I001 isort found an import in the wrong position` on line 5 and `I005 isort found an unexpected missing import` on line 6.

### How it should be

Maybe we should suppress C812 for multiline import?

### Flake8 version and plugins

{
  "dependencies": [],
  "platform": {
    "python_implementation": "CPython",
    "python_version": "3.10.6",
    "system": "Windows"
  },
  "plugins": [
    {
      "is_local": false,
      "plugin": "flake8-bandit",
      "version": "3.0.0"
    },
    {
      "is_local": false,
      "plugin": "flake8-broken-line",
      "version": "0.5.0"
    },
    {
      "is_local": false,
      "plugin": "flake8-bugbear",
      "version": "22.9.23"
    },
    {
      "is_local": false,
      "plugin": "flake8-comprehensions",
      "version": "3.10.0"
    },
    {
      "is_local": false,
      "plugin": "flake8-darglint",
      "version": "1.8.1"
    },
    {
      "is_local": false,
      "plugin": "flake8-debugger",
      "version": "4.1.2"
    },
    {
      "is_local": false,
      "plugin": "flake8-docstrings",
      "version": "1.6.0, pydocstyle: 6.1.1"
    },
    {
      "is_local": false,
      "plugin": "flake8-eradicate",
      "version": "1.4.0"
    },
    {
      "is_local": false,
      "plugin": "flake8-string-format",
      "version": "0.3.0"
    },
    {
      "is_local": false,
      "plugin": "flake8_commas",
      "version": "2.1.0"
    },
    {
      "is_local": false,
      "plugin": "flake8_isort",
      "version": "4.2.0"
    },
    {
      "is_local": false,
      "plugin": "flake8_quotes",
      "version": "3.3.1"
    },
    {
      "is_local": false,
      "plugin": "mccabe",
      "version": "0.6.1"
    },
    {
      "is_local": false,
      "plugin": "naming",
      "version": "0.13.2"
    },
    {
      "is_local": false,
      "plugin": "pycodestyle",
      "version": "2.8.0"
    },
    {
      "is_local": false,
      "plugin": "pyflakes",
      "version": "2.4.0"
    },
    {
      "is_local": false,
      "plugin": "rst-docstrings",
      "version": "0.2.7"
    },
    {
      "is_local": false,
      "plugin": "wemake_python_styleguide",
      "version": "0.17.0"
    }
  ],
  "version": "4.0.1"
}

### pip information

pip 22.2.2 from C:\Users\{username}\AppData\Local\pypoetry\Cache\virtualenvs\hrkg-VVSivBnA-py3.10\lib\site-packages\pip (python 3.10)
anyio @ file:///C:/Users/{username}/AppData/Local/pypoetry/Cache/artifacts/cd/3f/ae/baff749ce6cb4d7985e4142650605d2d30cb92eb418e2d121868e4413d/anyio-3.6.1-py3-none-any.whl
astor @ file:///C:/Users/{username}/AppData/Local/pypoetry/Cache/artifacts/09/5c/54/37b579cdfe7b1a219f41cb5d12fde801593f9c600ce5bee29dfe931626/astor-0.8.1-py2.py3-none-any.whl
attrs @ file:///C:/Users/{username}/AppData/Local/pypoetry/Cache/artifacts/b9/be/54/b0e6347a5b526e41f45aea10a61b5240586c74ee17e1d53c827b4d7288/attrs-22.1.0-py2.py3-none-any.whl
bandit @ file:///C:/Users/{username}/AppData/Local/pypoetry/Cache/artifacts/f0/49/b5/f7eef3f37396c309ab2b6498c81d9ab34d2d1734c1831218a4f2603023/bandit-1.7.4-py3-none-any.whl
click @ file:///C:/Users/{username}/AppData/Local/pypoetry/Cache/artifacts/63/f3/4c/2270b95f4d37b9ea73cd401abe68b6e9ede30380533cd4e7118a8e3aa3/click-8.1.3-py3-none-any.whl
colorama @ file:///C:/Users/{username}/AppData/Local/pypoetry/Cache/artifacts/c2/16/90/5bd8e7ca5465caa39fb4befd804a6c8a0a281f8f290c3387c173072719/colorama-0.4.5-py2.py3-none-any.whl
darglint @ file:///C:/Users/{username}/AppData/Local/pypoetry/Cache/artifacts/96/d1/6c/f02878c2e8c4678c3bddb09af981373d25ef06204e95910771b3c29aaf/darglint-1.8.1-py3-none-any.whl
docutils @ file:///C:/Users/{username}/AppData/Local/pypoetry/Cache/artifacts/49/06/62/0ad22e5dcbb2b9b2766963b30e3829d6081e9e68fccbab17e9e62b6c80/docutils-0.19-py3-none-any.whl
eradicate @ file:///C:/Users/{username}/AppData/Local/pypoetry/Cache/artifacts/60/be/db/8813e0e3bde9cd1fbe282ee0d5e3dcb5952e9a414b9877b9d7768bcc93/eradicate-2.1.0-py3-none-any.whl
fastapi @ file:///C:/Users/{username}/AppData/Local/pypoetry/Cache/artifacts/2d/cc/4b/b5a2b8f794a1b64df7460932b17a37b2b1ebfecc1cb26a333f8b4d91db/fastapi-0.85.0-py3-none-any.whl
flake8 @ file:///C:/Users/{username}/AppData/Local/pypoetry/Cache/artifacts/6b/d8/67/6516837c645c57af9cad7b290e1e6cbe92a126641fa84a5dde6495f3c3/flake8-4.0.1-py2.py3-none-any.whl
flake8-bandit @ file:///C:/Users/{username}/AppData/Local/pypoetry/Cache/artifacts/46/26/09/d23ed7dabffdc163aafabf20e9b18ffdd66126c3ac418b0f449444c88e/flake8_bandit-3.0.0-py2.py3-none-any.whl
flake8-broken-line @ file:///C:/Users/{username}/AppData/Local/pypoetry/Cache/artifacts/e4/4d/7d/8daa90d0f7de07b428f9e2c644050fa06e789e2fd52350717773cf6a77/flake8_broken_line-0.5.0-py3-none-any.whl
flake8-bugbear @ file:///C:/Users/{username}/AppData/Local/pypoetry/Cache/artifacts/bf/72/74/4daa59f623eb0a2a53aa697f827802a6e0b4ea9967a50775ce335aee26/flake8_bugbear-22.9.23-py3-none-any.whl
flake8-commas @ file:///C:/Users/{username}/AppData/Local/pypoetry/Cache/artifacts/38/ac/7d/f0ea8bbe164da39a1dcb9fe94ef5bcde6593bce7590888b188404d6afe/flake8_commas-2.1.0-py2.py3-none-any.whl
flake8-comprehensions @ file:///C:/Users/{username}/AppData/Local/pypoetry/Cache/artifacts/e4/8e/de/c1ce7c24b20f7a7a0d4b5002b262ea616d90443bebefae656437fa2d28/flake8_comprehensions-3.10.0-py3-none-any.whl
flake8-debugger @ file:///C:/Users/{username}/AppData/Local/pypoetry/Cache/artifacts/73/a8/b8/c1319aaef667b20f6e6a2e58ee729fb21e4bb93d6f8c042499f163a22e/flake8_debugger-4.1.2-py3-none-any.whl
flake8-docstrings @ file:///C:/Users/{username}/AppData/Local/pypoetry/Cache/artifacts/ed/64/32/5af4d87c0b9aea34bd8035c5c247a736d820fa8fe7f41a5249c06bf199/flake8_docstrings-1.6.0-py2.py3-none-any.whl
flake8-eradicate @ file:///C:/Users/{username}/AppData/Local/pypoetry/Cache/artifacts/42/a9/e4/7fd37abd98d9908ba54a8fed252a58fc4b2c7390b8ad82b4b9d47c50fc/flake8_eradicate-1.4.0-py3-none-any.whl
flake8-isort @ file:///C:/Users/{username}/AppData/Local/pypoetry/Cache/artifacts/f3/97/66/b135c4975df407a820b107b318efb693a5187d5fcc3f65ab092f3e6537/flake8_isort-4.2.0-py3-none-any.whl
flake8-polyfill @ file:///C:/Users/{username}/AppData/Local/pypoetry/Cache/artifacts/94/ed/18/a0bf327df9a14974261491aee203f5a18f9d02dcabf28e5fc17c480a98/flake8_polyfill-1.0.2-py2.py3-none-any.whl
flake8-quotes @ file:///C:/Users/{username}/AppData/Local/pypoetry/Cache/artifacts/2b/88/5e/1d0748067fdafe05339cfe02e65112cdfb05a761c70dda829e7ec9b80a/flake8-quotes-3.3.1.tar.gz
flake8-rst-docstrings @ file:///C:/Users/{username}/AppData/Local/pypoetry/Cache/artifacts/04/fc/99/eae85f42baab74e5ada2acfffb03953b9d01b7c228c3769bf5e43278ef/flake8_rst_docstrings-0.2.7-py3-none-any.whl
flake8-string-format @ file:///C:/Users/{username}/AppData/Local/pypoetry/Cache/artifacts/6f/04/a5/65c2427b493559b37bdf91e90f151729de6855625035d6f6b10a4c117d/flake8_string_format-0.3.0-py2.py3-none-any.whl
ghp-import @ file:///C:/Users/{username}/AppData/Local/pypoetry/Cache/artifacts/f1/35/49/f57df0715a342778564be838df3b63ca79206e4ccd18add537665c69c1/ghp_import-2.1.0-py3-none-any.whl
gitdb @ file:///C:/Users/{username}/AppData/Local/pypoetry/Cache/artifacts/60/7e/d5/cee7f2eee8ab34d772a60791e592b08b85f32a2644f242af55b7683912/gitdb-4.0.9-py3-none-any.whl
GitPython @ file:///C:/Users/{username}/AppData/Local/pypoetry/Cache/artifacts/99/c7/9b/54eaa24fada08ba17bd5fd2dcf1f9ae35c1d0d16434a66720f35aff7b1/GitPython-3.1.28-py3-none-any.whl
greenlet @ file:///C:/Users/{username}/AppData/Local/pypoetry/Cache/artifacts/15/1e/8b/7d9c44de04e819e7ad16873ff36e985dd2e614bafa2814ac98cef11c6c/greenlet-1.1.3-cp310-cp310-win_amd64.whl
idna @ file:///C:/Users/{username}/AppData/Local/pypoetry/Cache/artifacts/6c/c4/9a/4a40812e61d669fffc474987128ff48aab9be33cdddc5e5038bd96f4ac/idna-3.4-py3-none-any.whl
importlib-metadata @ file:///C:/Users/{username}/AppData/Local/pypoetry/Cache/artifacts/ea/30/f1/122927f5d13e9300164e3b942be74633397d23f24f7ad4ee4311634f0a/importlib_metadata-4.12.0-py3-none-any.whl
iniconfig @ file:///C:/Users/{username}/AppData/Local/pypoetry/Cache/artifacts/1d/67/0c/79706e7a41ef01be0b1a38aa3b4ddc9bd800a95ccd8d8dc7de4e090623/iniconfig-1.1.1-py2.py3-none-any.whl
isodate @ file:///C:/Users/{username}/AppData/Local/pypoetry/Cache/artifacts/c8/bb/f6/6689f9c69303fe358413a13ae731a962d78e50d897ea5e9070003ba5dd/isodate-0.6.1-py2.py3-none-any.whl
isort @ file:///C:/Users/{username}/AppData/Local/pypoetry/Cache/artifacts/12/e3/f1/e8e484225dcc659de6e4ac34a4ad996cb71f36119e54e71462cc7ad7e8/isort-5.10.1-py3-none-any.whl
Jinja2 @ file:///C:/Users/{username}/AppData/Local/pypoetry/Cache/artifacts/49/36/ae/943f6cd852641f7249acddef711eb97d0c9ed91d7f435c798b6d7041ca/Jinja2-3.1.2-py3-none-any.whl
Markdown @ file:///C:/Users/{username}/AppData/Local/pypoetry/Cache/artifacts/d0/a4/6e/be41805affb285f0d69e9684ea751cddcd71a6b9dbbf9b2373a33b1510/Markdown-3.3.7-py3-none-any.whl
MarkupSafe @ file:///C:/Users/{username}/AppData/Local/pypoetry/Cache/artifacts/63/4c/df/b6e3ad7ff5003118cde1283d2c91c6ebeba40e1c887d13aaa4d35f397c/MarkupSafe-2.1.1-cp310-cp310-win_amd64.whl
mccabe @ file:///C:/Users/{username}/AppData/Local/pypoetry/Cache/artifacts/a3/d3/68/f7f7115194f6fe8c487def3afa3b089a428f90650cf1b7be71afa83d12/mccabe-0.6.1-py2.py3-none-any.whl
mergedeep @ file:///C:/Users/{username}/AppData/Local/pypoetry/Cache/artifacts/70/94/ca/c320addc930721daba177fb325f66f4c680f55ce6458d2eae38e9ac7e0/mergedeep-1.3.4-py3-none-any.whl
mkdocs @ file:///C:/Users/{username}/AppData/Local/pypoetry/Cache/artifacts/fb/39/84/23731fa5812712d80a6805c12c9e9f861edc6a3dea2313065e0896e690/mkdocs-1.4.0-py3-none-any.whl
mkdocs-autorefs @ file:///C:/Users/{username}/AppData/Local/pypoetry/Cache/artifacts/cb/d8/ac/bbe58a32c41a97211a0d6c4074663a7db9bc79a490d56f13973fb4affc/mkdocs_autorefs-0.4.1-py3-none-any.whl
mkdocstrings @ file:///C:/Users/{username}/AppData/Local/pypoetry/Cache/artifacts/52/0e/6e/3596133ab8d0e7b0e15cb22cb8b8b91d04aee353321690274782b22eeb/mkdocstrings-0.19.0-py3-none-any.whl
packaging @ file:///C:/Users/{username}/AppData/Local/pypoetry/Cache/artifacts/d3/1f/90/3f2473250ebd055fe7f0856810677783949fc0d2e5f25bbb4dc00ad0a9/packaging-21.3-py3-none-any.whl
pbr @ file:///C:/Users/{username}/AppData/Local/pypoetry/Cache/artifacts/64/1c/ef/4f6348547068384a03e60ef9bf307ee778732fd7142b83f360de9e1c29/pbr-5.10.0-py2.py3-none-any.whl
pep8-naming @ file:///C:/Users/{username}/AppData/Local/pypoetry/Cache/artifacts/17/a6/c7/81acaff93c7ea6eeb262bb98287966c067735c4a219bd69b928d2cc00b/pep8_naming-0.13.2-py3-none-any.whl
pluggy @ file:///C:/Users/{username}/AppData/Local/pypoetry/Cache/artifacts/49/3e/55/309ec3705ba3309d817d531002c0a02f643d06c5d5a889ce43236130d5/pluggy-1.0.0-py2.py3-none-any.whl
py @ file:///C:/Users/{username}/AppData/Local/pypoetry/Cache/artifacts/71/fb/64/bfdc833c2055a893883f33deaac35ef2cbdd95d58031cfc69a630f32bf/py-1.11.0-py2.py3-none-any.whl
pycodestyle @ file:///C:/Users/{username}/AppData/Local/pypoetry/Cache/artifacts/67/1d/0b/aa6ec86495f3fc57e748e006fb36d7b4e65ddaf6de336617166ccad96d/pycodestyle-2.8.0-py2.py3-none-any.whl
pydantic @ file:///C:/Users/{username}/AppData/Local/pypoetry/Cache/artifacts/07/f2/4c/36ecab4b4d31d8f3377a0bf0775377c57d2a392e84d1d00ceae9719b79/pydantic-1.10.2-cp310-cp310-win_amd64.whl
pydocstyle @ file:///C:/Users/{username}/AppData/Local/pypoetry/Cache/artifacts/76/51/27/5fedbe9b911687054391ee48811ebde649f80f6f5833a6f2c00c896338/pydocstyle-6.1.1-py3-none-any.whl
pyflakes @ file:///C:/Users/{username}/AppData/Local/pypoetry/Cache/artifacts/ce/06/1e/5fd69e017a941ae783a7e4d3fb29abc5238f0026d7745b4dc26d7e3379/pyflakes-2.4.0-py2.py3-none-any.whl
Pygments @ file:///C:/Users/{username}/AppData/Local/pypoetry/Cache/artifacts/b5/d5/c4/6dbc728128a5cb37e5da9956d152c4eba518e3ff024956d0907394b234/Pygments-2.13.0-py3-none-any.whl
pymdown-extensions @ file:///C:/Users/{username}/AppData/Local/pypoetry/Cache/artifacts/6a/83/33/feeefc13f348bc2d3ae8e76974602645d6ff51b57a6fc6ce3df6985c68/pymdown_extensions-9.6-py3-none-any.whl
PyMySQL @ file:///C:/Users/{username}/AppData/Local/pypoetry/Cache/artifacts/c5/9a/90/ac7be251bb469ffd5947fac4a1114aa22299d37c0f7ee22525a4053e68/PyMySQL-1.0.2-py3-none-any.whl
pyparsing @ file:///C:/Users/{username}/AppData/Local/pypoetry/Cache/artifacts/e1/09/c0/6a6df3ef412eb19fcda1b353ea54e3b48f9c518348fadd0967a962cbb6/pyparsing-3.0.9-py3-none-any.whl
pypattyrn @ file:///C:/Users/{username}/AppData/Local/pypoetry/Cache/artifacts/ad/8b/8a/4300b687a911190c765e1c2692f6d7f12c766cec6c14bbaf109b70c047/pypattyrn-1.2.zip
pytest @ file:///C:/Users/{username}/AppData/Local/pypoetry/Cache/artifacts/18/f8/f9/0b374e1a1b65dce0ce9344fcd8f70645e767f1312f446372d8b107b46d/pytest-7.1.3-py3-none-any.whl
python-dateutil @ file:///C:/Users/{username}/AppData/Local/pypoetry/Cache/artifacts/39/9b/3b/576da7c854551d780528d75132c98882925e44de0aecf8a31a4bea379c/python_dateutil-2.8.2-py2.py3-none-any.whl
PyYAML @ file:///C:/Users/{username}/AppData/Local/pypoetry/Cache/artifacts/35/4d/45/b2142abd2ac771fdc795573de08522db0ff795d8e0a5c06f8baf9833e9/PyYAML-6.0-cp310-cp310-win_amd64.whl
pyyaml_env_tag @ file:///C:/Users/{username}/AppData/Local/pypoetry/Cache/artifacts/9d/65/01/91497406c47bf2cd172167f16613ef5b82e2fd1173e47ab5a2a067d9c4/pyyaml_env_tag-0.1-py3-none-any.whl
rdflib @ file:///C:/Users/{username}/AppData/Local/pypoetry/Cache/artifacts/f6/9b/5d/45f9ef7cd317b057e401693fe3074ced239569fca92c5edc8b7d2bba61/rdflib-6.2.0-py3-none-any.whl
restructuredtext-lint @ file:///C:/Users/{username}/AppData/Local/pypoetry/Cache/artifacts/81/bc/84/e389fe50ba4ac624a1658aed8cdcceb18a58f785c459c2c067edeb7af5/restructuredtext_lint-1.4.0.tar.gz
six @ file:///C:/Users/{username}/AppData/Local/pypoetry/Cache/artifacts/89/b2/f8/fd92b6d5daa0f8889429b2fc67ec21eedc5cae5d531ee2853828ced6c7/six-1.16.0-py2.py3-none-any.whl
smmap @ file:///C:/Users/{username}/AppData/Local/pypoetry/Cache/artifacts/c9/39/91/ed6e3da1cbfcb68393f3640f3d84c7afd8574257495adb1fe2615bbeaa/smmap-5.0.0-py3-none-any.whl
sniffio @ file:///C:/Users/{username}/AppData/Local/pypoetry/Cache/artifacts/c9/3f/3e/8e46c9457f352d5bba461491356dfede8138f565b9e1015719034b36a1/sniffio-1.3.0-py3-none-any.whl
snowballstemmer @ file:///C:/Users/{username}/AppData/Local/pypoetry/Cache/artifacts/c4/88/1b/a9b3e7fb8e193ad3ed607c754eb234b06cbd6f0a43289406a2dc4eddbf/snowballstemmer-2.2.0-py2.py3-none-any.whl
SQLAlchemy @ file:///C:/Users/{username}/AppData/Local/pypoetry/Cache/artifacts/d1/7f/79/b389f224dc7494739da0dfcabc3ec1b4580dcc34ebb3870c0dd36d2da4/SQLAlchemy-1.4.41-cp310-cp310-win_amd64.whl
starlette @ file:///C:/Users/{username}/AppData/Local/pypoetry/Cache/artifacts/5f/9f/8b/92d08d810170b0d38d0ebbb13b8c8a7b292fa829bb433f848d30764492/starlette-0.20.4-py3-none-any.whl
stevedore @ file:///C:/Users/{username}/AppData/Local/pypoetry/Cache/artifacts/0b/0c/e7/bfc72248deb68adcc02e9624ecacab9cd9f6b8bbe9b519ee0e1f11deb8/stevedore-4.0.0-py3-none-any.whl
tomli @ file:///C:/Users/{username}/AppData/Local/pypoetry/Cache/artifacts/62/12/b6/6db9ebb9c8e1a6c5aa8a92ae73098d8119816b5e8507490916621bc305/tomli-2.0.1-py3-none-any.whl
typing_extensions @ file:///C:/Users/{username}/AppData/Local/pypoetry/Cache/artifacts/5a/62/d2/6e114f8e6602ecb5e40ed745c522ed05ef05b697e5432317e3de8b6bcd/typing_extensions-4.4.0-py3-none-any.whl
watchdog @ file:///C:/Users/{username}/AppData/Local/pypoetry/Cache/artifacts/d1/2f/8c/931227478371e670ff64027717fbe312520de12dbb4c1d108469e8801d/watchdog-2.1.9-py3-none-win_amd64.whl
wemake-python-styleguide @ file:///C:/Users/{username}/AppData/Local/pypoetry/Cache/artifacts/c4/2e/d5/69b1b4be4f4d564cc062d0b1795206aa75ae86ddd5751d63cc5d91e187/wemake_python_styleguide-0.17.0-py3-none-any.whl
zipp @ file:///C:/Users/{username}/AppData/Local/pypoetry/Cache/artifacts/6b/51/e2/473bdd21a093cb9b26ed4495aed0be9a948ef966a3ffe804aa300b144d/zipp-3.8.1-py3-none-any.whl

### OS information

Edition	Windows 10 Enterprise
Version	21H2
Installed on	‎2/‎9/‎2022
OS build	19044.2006
Experience	Windows Feature Experience Pack 120.2212.4180.0

**Comment by sobolevn** [comment]
You can:
- Use `--profile=wemake` to get the best experience
- Use [`--include_trailing_comma` together with `--multi-line-output`](https://github.com/wemake-services/wemake-python-styleguide/blob/master/styles/isort.toml#L16)

Please, reopen if it does not work for you!
